lib/linux_admin/security/common.rb

```diff
@@ -0,0 +1,14 @@
+module LinuxAdmin
+  class Security
+    module Common
+      def replace_config_line(new_line, rep_regex, file_text)
+        new_text = file_text.gsub!(rep_regex, new_line)
```

Comment:
I could do

```ruby
String.new(file_text).gsub!(rep_regex, new_line)
```

because the caller of this method should expect a return, rather than using the passed in object.  I think that will lead to less confusion as I'm only using this version because of the return behavior. What do you think?